src/SFA.DAS.AdminService.Web/Views/Search/Select.cshtml

```diff
@@ -183,7 +184,7 @@ else
                             @CertificateStatus.GetDisplayName(Model.Learner.CertificateStatus)
                         </dd>
                     </div>
-                    @if (!string.IsNullOrEmpty(Model.DateStatusTitle))
+                    @if (Model.UpdatedStatusDate != null)
```

Comment:
If will always have a date according to the logic above - so what is the if statement for - which AC?